modules/space/kepler.cpp

```diff
@@ -266,15 +266,20 @@ namespace {
             epoch.end(),
             [](char c) { return c == '-'; }
         );
-        const std::string format = (nDashes == 2) ? "{:4}-{:2}-{:2}{}" : "{:4}{:2}{:2}{}";
-        auto res = scn::scan<int, int, int, double>(e, scn::runtime_format(format));
+        const std::string format = (nDashes == 2) ? "{:4d}-{:2d}-{}" : "{:4d}{:2d}{}";
+        auto res = scn::scan<int, int, double>(e, scn::runtime_format(format));
```

Comment:
```suggestion
        auto res = scn::scan<int, int, double>(
            e,
            (nDashes == 2) ? "{:4d}-{:2d}-{}" : "{:4d}{:2d}{}"
        );
```

Reply:
This would skip the extra complexity of the `runtime_format`

Reply:
I tried this, but got error:
`the value of ‘nDashes’ is not usable in a constant expression`
so I'll keep it as-is.